tools/fuzzer/expr_gen.cc

```diff
@@ -628,6 +626,14 @@ std::optional<Expr> ExprGenerator::gen_address_of_expr_impl(
 
   TypeConstraints new_type_constraints =
       constraints.type_constraints().allowed_to_point_to();
+
+  if (!new_type_constraints.satisfiable()) {
+    // It is possible that type constraints become unsatisfiable after the
+    // `allowed_to_point_to()` is performed (e.g. `void*` can't become `void`
+    // since `void` expressions are invalid).
```

Comment:
Does this already happen or is it introduced in this pull request? IIUC right now `allowed_to_point_to` doesn't do the pruning of `void` from `void*` (it only forbids `void` if pointer constraint is `AnyType`).